### DISCUSSION TITLE
Where can I find the in-browser utility code?

### DISCUSSION THREAD

**Discussion by ranshemt** [moov-io/ach#1122] [Q&A] [answered]
Hello,

Where can find the code of the [in-browser ACH file parser](https://github.com/moov-io/ach#in-browser-ach-file-parser)? I understand that it uses web assembly and no server side involved.
Do you have instructions to create such a viewer myself?

**Comment by adamdecaf** [comment, marked as answer]
It's in the [cmd/webui package](https://github.com/moov-io/ach/tree/master/cmd/webui). We release a [moov/ach-webui docker image](https://hub.docker.com/r/moov/ach-webui) you could use directly. This container is what powers oss.moov.io/ach/

**Reply by ranshemt** [reply]
Thanks!
